include/envoy/network/udp_packet_writer_handler.h

```diff
@@ -0,0 +1,116 @@
+#pragma once
+
+#include <cstdint>
+#include <memory>
+
+#include "envoy/api/io_error.h"    // IoCallUint64Result
+#include "envoy/buffer/buffer.h"   // Buffer
+#include "envoy/network/address.h" // Address
+#include "envoy/network/socket.h"  // Socket
```

Comment:
nit: these comments will get out of date, I would remove

Reply:
Updated these coments to clear this here!